internal/common/time.go

```diff
@@ -25,55 +25,35 @@
 package common
 
 import (
-	"math"
+	"time"
 )
 
-// Int32Ceil return the int32 ceil of a float64
-func Int32Ceil(v float64) int32 {
-	return int32(math.Ceil(v))
-}
-
-// Int64Ceil return the int64 ceil of a float64
-func Int64Ceil(v float64) int64 {
-	return int64(math.Ceil(v))
-}
-
-// MinInt64 returns the smaller of two given int64
-func MinInt64(a, b int64) int64 {
-	if a < b {
-		return a
+// TimeValue dereference time in a safe way.
+func TimeValue(t *time.Time) time.Time {
+	if t == nil {
+		return time.Time{}
```

Comment:
On server I am treating this as `time.Unix(0, 0)` - https://github.com/temporalio/temporal/blob/master/common/primitives/timestamp/time.go#L192

Reply:
Yes, I am changing it there also. I think we need to use `time.Time{}` everywhere as `nil` replacement.